documentation/0.14.0-SNAPSHOT/html/developer-guide/index.html

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I know we're not actually going to publish SNAPSHOT documentation using this mechanism, but it was a useful exercise.